talent/one_on_one_guide.md

```diff
@@ -0,0 +1,98 @@
+# source{d} One-On-One’s Guide
+
+Below is the guide we use at source{d} for one-on-one meetings between you and your manager. They are guiding questions and it's okay if someone choses to adapt them.
+
+## Schedule:
+
+* One-on-one's are to be held at a minimum every 3 months
+
+## General:
+
+* Refer back to your private notes & report of the previous month before meeting the person
+
+* Make sure any "company news" has been understood by the team and any doubts addressed 	
+
+* Communication inside the teams and inside the company can always be improved
+
+## Cardinal rules:
+
+* No matter in what part below, if something comes up you believe should stay private between you and your team member or has the expectation of privacy, keep it private!
+
+* Take notes
+
+* No phone (full attention)
+
+* Be on time!
+
+* Body language is important (no arms crossed, no sitting in opposite sides of table)
+
+* Don't look at the time (set an alarm after 1h if needed)
+
+* They need to be in the conference room or in a quiet environment (not a bar)
+
+* Set the one-on-one’s in everyone’s calendar 
+
+
+## Suggested Template:
+
+Starting the meeting please let the person know that anything they consider private will be kept private between the two of you but general feedback on the company, performance related, improvement suggestions and career development may be discussed with members of the management team or other leads.
+
+### Part I. Company feedback
+
+* Start with gathering feedback on the company:
+
+    * What do you believe is source{d}’s strategy right now?
+
+    * Where do you have doubts about the company?
+
+    * How do you feel our culture is evolving?
+
+    * Take note of information that seems unknown to people that needs to be communicated better
+
+### Part II. Self-assessment
+
+* Please let me know what you believe you’ve done well at source{d} in the last month?
+
+* Please let me know where you believe you can still improve? (it is important this becomes part of the action plan on how to improve)
+
+* If you could rate your happiness from 1-10? If not 10, why not and what can we do to improve it 
+
+### Part III. Team improvement
+
+* What do you believe we can do better as a team? 
+
+### Part IV. Feedback on performance
+
+* Praise first: what was well done and why you think so. It is important to give concrete examples here of deliverables/projects etc. you believe went well
+
+* Downsides: always mention the negative followed by actions to improve it. Before your meeting be sure you have prepared an action plan (set of bullet points on how you will together improve things) to tackle the downsides and **follow up** on it until the following one-on-one. 
+
+### Part V. Career Development
```

Comment:
It's more about following up. Someone says I want to be "X in 5 years". Then you discuss if during the last 3 months they did anything to help get there. I should clarify this.